github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -1043,6 +1043,95 @@ presubmits:
           path: /var/log/audit
           type: Directory
         name: audit
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 4h0ms
+      grace_period: 5m0s
```

Need to add in order to handle getting the kubeconfig
```suggestion
    extra_refs:
    - base_ref: main
      org: kubevirt
      repo: project-infra
```